core/templates/pages/preferences-page/preferences-page.component.ts

```diff
@@ -128,6 +128,7 @@ export class PreferencesPageComponent {
   }
 
   saveUserBio(userBio: string): void {
+    this.registerBioChanged();
```

Comment:
It looks like `_saveDataItem` adds a listener for the page unload event, so we probably don't need the `registedBioChanged` method call here (it does the same thing). Please check if this method is used anywhere, if not we can delete the method altogether.

Reply:
Yes, I was wondering the same thing. Why is the same thing done twice? I thought that there must be a reason and didn't remove it.